recipe/meta.yaml

```diff
@@ -1,36 +1,56 @@
 {% set version = "0.9.0" %}
 
 package:
-  name: geopandas
+  name: geopandas-suite
   version: {{ version }}
 
 source:
   url: https://github.com/geopandas/geopandas/releases/download/v{{ version }}/geopandas-{{ version }}.tar.gz
   sha256: 63972ab4dc44c4029f340600dcb83264eb8132dd22b104da0b654bef7f42630a
 
 build:
-  number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-  run:
-    - python >=3.6
-    - pandas >=0.24
-    - shapely
-    - fiona
-    - pyproj >=2.2.0
-    - rtree
-
-test:
-  imports:
-    - geopandas
-    - geopandas.io
-  files:
-    - test_data
+  number: 1
+  noarch: python
+
+outputs:
+  - name: geopandas-base
+    script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
Sadly conda-build `script` in multiple outputs does work like `script` in a single output. We must point it to a [script file like  this](https://github.com/conda-forge/importlib_metadata-feedstock/blob/master/recipe/meta.yaml).

Reply:
Thanks for the pointer! That's working now